app/editor/src/components/modal/styled/Modal.tsx

```diff
@@ -25,7 +25,7 @@ export const Modal = styled(Col)`
   }
 
   .button-row {
-    margin-top: 10%;
+    margin-top: 5px;
```

Comment:
I am trying to only use `em/rem` instead of `px `as it is best practice. This should be fine but just for future! I am guilty of using px all the time 😛